src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionCoordinatorModelTests.swift

```diff
@@ -30,7 +30,7 @@ class ExposureSubmissionCoordinatorModelTests: CWATestCase {
 				client: client,
 				appConfiguration: appConfiguration,
 				boosterNotificationsService: BoosterNotificationsService(
-					rulesDownloadService: RulesDownloadService(store: store, client: client)
+					rulesDownloadService: RulesDownloadService(restServiceProvider: RestServiceProviderStub.fake())
```

Comment:
is it possible to omit `RestServiceProviderStub`?

```suggestion
					rulesDownloadService: RulesDownloadService(restServiceProvider: .fake())
```

Reply:
Nice finding 👍